src/engine/source/router/src/runtimeEnvironment.cpp

```diff
@@ -1,125 +1,45 @@
 #include <router/runtimeEnvironment.hpp>
 
-#include "base/parseEvent.hpp"
+#include <base/parseEvent.hpp>
 #include <logging/logging.hpp>
 #include <rxbk/rxFactory.hpp>
 #include <utils/getExceptionStack.hpp>
 
-// TODO: Refactor how we handle queue flooding and environments down
-std::atomic_bool g_envDown {true};
 
 namespace router
 {
-constexpr auto WAIT_DEQUEUE_TIMEOUT_USEC = 1 * 1000000;
 
-std::optional<base::Error>
-RuntimeEnvironment::build(std::shared_ptr<builder::Builder> builder)
+std::optional<base::Error> RuntimeEnvironment::build(std::shared_ptr<builder::Builder> builder)
 {
-    if (m_isRunning)
+    if (m_controller)
     {
-        return base::Error {
-            "Engine runtime environment: Environment is already running."};
-    }
-
-    if (m_environments.size() > 0)
-    {
-        return base::Error {"Engine runtime environment: Environment is already built"};
+        return base::Error {fmt::format("Environment '{}' is already built.", m_asset)};
```

Comment:
base::Error messages are not supposed to end with a "." (dot) as it is printed on a higher level. Please, fix every case.